nox.py

```diff
@@ -44,10 +44,13 @@ def check_requirements(session):
 
 @nox.session
 def lint(session):
+    session.interpreter = 'python3'
     session.install('flake8', 'flake8-import-order')
     session.run(
         'flake8',
         '--import-order-style=google',
+        ('--application-import-names='
```

Comment:
Done, although it needs to be like:

    '--application-import-names',
    'gen_dockerfile,local_cloudbuild,validation_utils',

Otherwise flake8 doesn't know when that list stops and the next one starts.